.golangci.yml

```diff
@@ -114,23 +115,27 @@ linters:
     - unparam
     - unused
     - varcheck
+    - testpackage
     - whitespace
+    - wsl
+    - nlreturn
+    - nestif
+    - gocognit
+    - gci
+    - forbidigo
+    - errorlint
+    - revive
```

Comment:
since we're adding revive we could just remove [golint](https://github.com/ZupIT/horusec-devkit/pull/126/files#diff-6179837f7df53a6f05c522b6b7bb566d484d5465d9894fb04910dd08bb40dcc9R97)